cou/apps/app.py

```diff
@@ -32,11 +32,7 @@
 )
 from cou.steps import UpgradeStep
 from cou.utils.app_utils import upgrade_packages
-from cou.utils.juju_utils import (
-    async_get_status,
-    async_set_application_config,
-    async_upgrade_charm,
-)
+from cou.utils.juju_utils import get_status, set_application_config, upgrade_charm
```

Comment:
I prefer to use all juju_utils function with a common prefix such as `jutils` or `utils` or `ut` it makes fırther refactoring easier

Reply:
I will use `from cou.utils import juju_utils` everywhere.